File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.java
class Solution {
    public int solution(int num, int k) {
        int answer = -1;
        int cnt = 0;
        
        while (num > 0) {
            int tmp = num % 10;
            if (tmp == k) {
                answer = cnt;
            }
            num /= 10;
            cnt++;
        }
        if (answer != -1) {
            answer = cnt - answer;
        }
        return answer;
    }
}

File: 프로그래머스/0/120904. 숫자 찾기/숫자 찾기.py
def solution(num, k):
    answer = -1
    cnt = 0

    while num > 0:
        tmp = num % 10
        if tmp == k:
            answer = cnt
        num //= 10
        cnt += 1
    if answer != -1:
        answer = cnt - answer
    return answer